views/js/controller/creator/views/test.js

```diff
@@ -64,6 +66,11 @@ define([
         //add feature visibility properties to testModel
         featureVisibility.addTestVisibilityProps(testModel);
 
+        const {featureFlags} = context;
+        if(featureFlags.FEATURE_FLAG_UNIQUE_NUMERIC_QTI_IDENTIFIER || testModel.translation) {
```

Comment:
**chore:** Code style: is prettier activated and properly configured?

Reply:
Thanks. Now I have configured the VSCode to use common config for all the projects.